cpp.md

```diff
@@ -274,8 +278,57 @@ typedef data-type name;
 typedef unsigned int distance;  // typedef of int 
 ```
 
-## Vectors
-Vectors are same as dynamic arrays. They will be resized on element inseertions & deletions.
+## Sort one-line
```

Comment:
This is a very specific use-case, does not belong in Cheat Sheets, please wite this as a post instead

Reply:
Hey @onecompiler-ops Please reply my question ! I want to contribute to this repo.